src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const properties = sourceString.split(';');
+
+  return properties
+    .map((property) => property.trim())
+    .filter((property) => property.length > 0)
+    .reduce((prev, property) => {
+      const [key, value] = property.split(':');
+
+      return {
+        ...prev,
+        [key.trim()]: value.trim(),
+      };
+    }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Please adjust the indentation of the chained methods to align with the good example provided in the checklist.